Objects/Converters/ConverterAutocadCivil/ConverterAutocadCivilShared/ConverterAutocadCivil.Civil.cs

```diff
@@ -189,6 +185,28 @@ public CivilAlignment AlignmentToSpeckle(CivilDB.Alignment alignment)
 
     return speckleAlignment;
   }
+
+  private List<Base> DesignSpeedsToSpeckle(DesignSpeedCollection designSpeeds)
+  {
+    List<Base> speckleDesignSpeeds = new();
+
+    foreach (DesignSpeed designSpeed in designSpeeds)
+    {
+      Base speckleDesignSpeed = new();
```

Comment:
Only consideration here would be we may want to just have an object type for this instead? Once we decide it's `Base`, undoing this will be a PITA 😅